cli/validator.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"math/big"
+	"strconv"
+
+	eigenpodproofs "github.com/Layr-Labs/eigenpod-proofs-generation"
+	"github.com/Layr-Labs/eigenpod-proofs-generation/cli/onchain"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/fatih/color"
+)
+
+func RunValidatorProof(ctx context.Context, eigenpodAddress string, eth *ethclient.Client, chainId *big.Int, beaconClient BeaconClient, out *string, owner *string) {
+	latestBlock, err := eth.BlockByNumber(ctx, nil)
+	PanicOnError("failed to load latest block", err)
```

Comment:
From the docs on this method, seems like we can/should use `HeaderByNumber` to reduce this from 2 requests to 1